src/builder.js

```diff
@@ -36,6 +37,7 @@ const FormBuilder = ({
   isoCode,
   ...props
 }) => {
+  console.log(schema.validate(form))
```

Comment:
Maybe it would be better if instead of logging an error it will throw in case the schema is incorrect, what do you think?